libs/island-ui/core/src/lib/Checkbox/Checkbox.tsx

```diff
@@ -92,7 +94,7 @@ export const Checkbox = ({
           />
         </div>
         <span className={styles.labelText}>
-          <Text>{label}</Text>
+          <Text variant={strong ? 'h5' : 'default'}>{label}</Text>
```

Comment:
Why change the `variant` instead of using the `fontWeight` prop? The `strong` prop name may be a bit misleading in that case.

Reply:
agree! just followed figma where h5 was being used.. this is way less misleading